packages/playwright/src/reporters/junit.ts

```diff
@@ -33,11 +33,13 @@ class JUnitReporter extends EmptyReporter {
   private outputFile: string | undefined;
   private resolvedOutputFile: string | undefined;
   private stripANSIControlSequences = false;
+  private includeProjectInTestName = false;
 
-  constructor(options: { outputFile?: string, stripANSIControlSequences?: boolean } = {}) {
+  constructor(options: { outputFile?: string, stripANSIControlSequences?: boolean, includeProjectInTestName?: boolean } = {}) {
     super();
     this.outputFile = options.outputFile || reportOutputNameFromEnv();
     this.stripANSIControlSequences = options.stripANSIControlSequences || false;
+    this.includeProjectInTestName = options.includeProjectInTestName || false;
```

Comment:
We have a new guidelines - all the reporter properties should be configurable via env. That way they can be merged w/o additional config.

Reply:
Sounds good, I will follow up with env vars for all the options.